src/main/java/ua/softserveinc/tc/service/BookingService.java

```diff
@@ -43,12 +39,57 @@
 
     List<BookingDto> updateRecurrentBookings(BookingDto recurrentBookingDtos);
 
+    /**
+     * Gets the all bookings by given start and end date, booking states.
+     *
+     * @param startDate the start date
+     * @param endDate the end date
+     * @param bookingStates the given booking states
+     * @return the list of appropriate bookings
+     */
+    List<Booking> getBookings(Date startDate, Date endDate, BookingState... bookingStates);
+
+    /**
+     * Gets the all bookings by given dates, user, booking states.
+     *
+     * @param dates the given dates
+     * @param user the given user
+     * @param bookingStates the given booking states
+     * @return the list of appropriate bookings
+     */
+    List<Booking> getBookings(Date[] dates, User user, BookingState... bookingStates);
```

Comment:
Do we really need an array of dates in all this functions? Why not to use startDate and endDate parameters?